example/src/api/secondProvider/posts.ts

```diff
@@ -55,8 +55,26 @@ export const postsApi = {
         const url = endpoint(clientNumber, postId);
         return axios.delete(url);
     },
+
+    /**
+     * @pact
+     * @pact-path /api/clients/10/posts/25/status
+     * @pact-method GET
+     */
+    getStatusOfPost: async function ({clientNumber, postId}: {clientNumber: string; postId: string}) {
```

Comment:
Could you rename it to not mix `get` and `post` in one function?

Reply:
what do you mean ?